bpdv3/Assets/Scripts/Floor/TileProperties.cs

```diff
@@ -20,7 +20,7 @@ public class TileProperties : MonoBehaviour
 
         #region Public & Protected variables
 
-        public int tileWithToken;
+        public int tileWithToken = 0;
```

Comment:
i added this to remove an error message from the console also, not sure if it was needed though thinking about again but keeping it does make the variable defined which doesn't hurt.

Reply:
That is why you initialize it to null